source/MRMesh/MRObjectMesh.h

```diff
@@ -48,6 +48,9 @@ class MRMESH_CLASS ObjectMesh : public ObjectMeshHolder
     using MeshChangedSignal = boost::signals2::signal<void( uint32_t mask )>;
     MeshChangedSignal meshChangedSignal;
 
+    /// reflects the mesh from a given plane
+    MRMESH_API void Mirror( const Plane3f& plane );
```

Comment:
better add this to `Mesh` struct, and we don't name member functions with capital latter by code style